charts/ocis/values.yaml

```diff
@@ -907,191 +1890,412 @@ services:
     nodeSelector: {}
+    # @schema
+    # type: [string, null]
+    # required: false
+    # @schema
     # -- Per-service priorityClassName configuration. Overrides the default setting from `priorityClassName` if set.
-    priorityClassName: ""
+    priorityClassName:
+    # @schema
+    # $ref: schema/templates.json#/podDisruptionBudgetTemplate
+    # @schema
     # -- Per-service PodDisruptionBudget. Overrides the default setting from `podDisruptionBudget` if set.
     podDisruptionBudget: {}
+    # @schema
+    # $ref: schema/templates.json#/autoscalingTemplate
+    # @schema
     # -- Per-service autoscaling. Overrides the default setting from `autoscaling` if set.
     autoscaling: {}
-    # -- Affinity settings for the approvider service.
+    # @schema
+    # $ref: schema/templates.json#/affinityTemplate
+    # @schema
+    # -- Per-service affinity.
     affinity: {}
-    # -- Configuration for the appprovider service, every other service will have the same configuration.
-    # The value of this key will be passed to the affinity setting of the pod as documented here:
-    # https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
-    # This example will ensure that the pods will only be scheduled on nodes with the label `topology.kubernetes.io/zone`
-    # set to `eu-west-1` or `eu-east-1`.
-    # nodeAffinity:
-    #   requiredDuringSchedulingIgnoredDuringExecution:
-    #     nodeSelectorTerms:
-    #     - matchExpressions:
-    #       - key: topology.kubernetes.io/zone
-    #         operator: In
-    #         values:
-    #         - eu-east-1
-    #         - eu-west-1
-    #
-    # The following example will make sure that these pods will never be scheduled on the same node.
-    # podAntiAffinity:
-    #   requiredDuringSchedulingIgnoredDuringExecution:
-    #     - labelSelector:
-    #         matchExpressions:
-    #           - key: app
-    #             operator: In
-    #             values:
-    #               - appprovider
-    #       topologyKey: "kubernetes.io/hostname"
-    #
+    # @schema
+    # $ref: schema/templates.json#/labelsTemplate
+    # @schema
     # Do note that the value will be different for each service.
     # -- Per-service custom labels
     extraLabels: {}
     # Per-service image configuration. Overrides the default setting from `image` if set.
     image:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image repository
-      repository: ""
+      repository:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image tag.
-      tag: ""
+      tag:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image sha / digest (optional).
-      sha: ""
+      sha:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
```

Comment:
We should add an enum here with the three values for `pullPolicy`

Reply:
same as https://github.com/owncloud/ocis-charts/pull/811/files#r1921904014